bin/setup_stack

```diff
@@ -105,7 +111,7 @@ for index in 1; do
     cp -r topo/$mode/forch inst/$controller/
     cp -r topo/$mode/faucet inst/$controller/
     docker kill $controller || true
-    bin/run_faucet $local $index $((6000 + index)) $((8000 + index))
+    bin/run_faucet $local $fot $index $((6000 + index)) $((8000 + index))
```

Comment:
as per other comment, rather than $fot here as an option to run_faucet, it should look something like:
```
if [ -n "$fot" ]; then
  bin/run_controller $local $index ... (whatever options)
else
  ... (the stuff that's currently done with bin/run_faucet)
fi
```

Reply:
Done.